quartz-core/src/main/java/org/quartz/core/QuartzSchedulerThread.java

```diff
@@ -27,6 +27,8 @@
 import org.quartz.SchedulerException;
 import org.quartz.Trigger;
 import org.quartz.Trigger.CompletedExecutionInstruction;
+import org.quartz.impl.jdbcjobstore.JobStoreSupport;
```

Comment:
I see it more like an integration-with, rather than dependency-on, but ok, will remove.